tldw_Server_API/tests/Flashcards/test_flashcards_templates_api.py

```
@@ -0,0 +1,276 @@
+from tldw_Server_API.tests.test_config import TestConfig
+
+AUTH_HEADERS = {"X-API-KEY": TestConfig.TEST_API_KEY}
+
+
+def _build_test_app() -> FastAPI:
+    app = FastAPI()
+    app.include_router(flashcards_router, prefix="/api/v1")
+    return app
+
+
+@pytest.fixture(scope="function")
```

Ruff (0.15.10)

[warning] `scope='function'` is implied in `@pytest.fixture()`

Remove implied `scope` argument

(PT003)